go.mod

```diff
@@ -2,10 +2,13 @@ module github.com/kubearmor/kubearmor-client
 
 go 1.16
 
+replace github.com/kubearmor/KubeArmor/protobuf => github.com/daemon1024/KubeArmor/protobuf v0.0.0-20211216122055-cfd556a829ef
```

Comment:
Will remove once https://github.com/kubearmor/KubeArmor/pull/536 is merged and package hash is updated